mvvm/model/tagrow.h

```diff
@@ -22,6 +22,13 @@ struct CORE_EXPORT TagRow
 {
     std::string tag = {};
     int row = -1;
+
+    static TagRow defaultAppend();
+
+    static TagRow append(const std::string& tag_name);
+
+    bool operator==(const TagRow& other) const;
+    bool operator!=(const TagRow& other) const;
```

Comment:
Usually such operators are declared as global functions (to allow for implicit conversions).
Could you please check if the following comparison works properly?

```
TagRow tagrow = {"abc", 23};
EXPECT_TRUE({"abc", 23} == tagrow);
EXPECT_FALSE({"abc", 13} == tagrow);
```

Or was it deliberate?

Reply:
Thanks, boolean was moved to global